client/my-sites/people/delete-user/style.scss

```diff
@@ -15,11 +15,23 @@
 	margin: 0 3px 0 8px;
 }
 
-.delete-user__remove-user .gridicon {
-	margin: -2px 4px 0 0;
-	vertical-align: middle;
-}
-
-.delete-user__remove-user {
-	cursor: pointer;
-}
+.delete-user__multisite {
+	.button.is-borderless {
+		&.delete-user__remove-user {
```

Comment:
A simple selector should be specific enough:
```
.delete-user__remove-user.button.is-borderless
```
The `.delete-user__multisite` wrapper is not necessary, as the BEM `.delete-user__remove-user` class names are designed to be unique. And the three levels of nesting are not necessary either.

Reply:
Full disclosure, I was uncertain and took me a while to figure out how to target `.delete-user__remove-user` and the gridicon without the wrapper. Did not realise it would be this simple. :) 

I have updated with your suggestion in b220c529e4933ecdbd6dfdb0c7f2a5289225394d. ✅